src/main/java/uk/gov/hmcts/reform/sscs/config/jms/JsonMessageConvertor.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.sscs.config.jms;
```

Comment:
this is a service, not a config, can you move it?

Reply:
maybe to into sscs.jms?

Reply:
Ah you are right I will move it to JMS file.